packages/voronoi/src/Mesh.tsx

```diff
@@ -102,13 +125,55 @@ export const Mesh = <Datum,>({
 
     const handleClick = useCallback(
         (event: MouseEvent<SVGRectElement>) => {
-            const [index, node] = getIndexAndNodeFromEvent(event)
+            const [index, node] = getIndexAndNodeFromMouseEvent(event)
             setCurrentIndex(index)
             if (node) {
                 onClick?.(node, event)
             }
         },
-        [getIndexAndNodeFromEvent, setCurrentIndex, onClick]
+        [getIndexAndNodeFromMouseEvent, setCurrentIndex, onClick]
+    )
+
+    const handleTouchStart = useCallback(
+        (event: TouchEvent<SVGRectElement>) => {
+            const [index, node] = getIndexAndNodeFromTouchEvent(event)
+            if (touchCrosshair) {
+                setCurrentIndex(index)
+            }
+            if (node) {
+                onTouchStart?.(node, event)
+            }
+        },
+        [getIndexAndNodeFromTouchEvent, setCurrentIndex, onTouchStart]
+    )
+
+    const handleTouchMove = useCallback(
```

Comment:
`touchCrosshair` dependency is missing.

Reply:
Updated in https://github.com/WilliamABradley/nivo/commit/f2e5fbb6007e02bbf0a7ea0f4f363916fe7ee138